install.md

```diff
@@ -378,8 +378,7 @@ cat /etc/containers/policy.json
 ## Vendoring
 
 Buildah uses Go Modules for vendoring purposes.  If you need to update or add a vendored package into Buildah, please follow this proceedure:
- * Enter into your sandbox `src/github.com/containers/buildah` and ensure that he GOPATH variable is set to the directory prior as noted above.
- * `export GO111MODULE=on`
```

Comment:
That's still needed.

Reply:
some of it should be reverted.  The change to the line above the export should stay (he -> the typo fix)

Reply:
Yes, I kept the typo fix. :)